ci-operator/config/netobserv/flowlogs-pipeline/netobserv-flowlogs-pipeline-main.yaml

```diff
@@ -18,14 +34,32 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.16"
+      version: "4.17"
```

Comment:
This version denotes which OCP version the cluster is built with? If so why 4.17 and not 4.18?

Reply:
Good point, I think this was due to ClusterPool, where currently our ClusterPool is setup for 4.17, but here for FLP and ebpf-agent we're not using for clusterpool, let me try changing here for 4.18 do rehearsals again.